frontend/packages/metal3-plugin/src/plugin.ts

```diff
@@ -123,6 +123,17 @@ const plugin: Plugin<ConsumedExtensions> = [
         ).then((m) => m.default),
     },
   },
+  {
+    type: 'Page/Route',
+    properties: {
+      exact: true,
+      path: ['/k8s/cluster/nodes/:name'],
```

Comment:
@jtomasek @spadgett FYI, for now the `RoutePage` extension doesn't support gating by feature flags, so this will always override ^^ route, assuming that Metal3 plugin is part of Console build.